templates/default/nginx.conf.erb

```diff
@@ -0,0 +1,34 @@
+user www-data;
```

Comment:
for this i took the default config that ships with nginx v1.4.6 and removed all the commented out sections.